docs/GRAPHQL.md

```diff
@@ -5,20 +5,23 @@
 ### `[GET|POST] /api/v2.2/graphql`
 
 Accepts `GET` or `POST` requests to perform GraphQL queries. See:
-- https://www.graphql-java.com/tutorials/getting-started-with-spring-boot/
-- https://graphql.org/learn/serving-over-http/
+- [https://www.graphql-java.com/tutorials/getting-started-with-spring-boot/](Getting Started)
```

Comment:
I guess it has to be in `[text](link)` format?

![Screenshot from 2022-09-20 11-21-22](https://user-images.githubusercontent.com/68053619/191298535-40d460ae-a8c5-48ad-a2ab-dd3de1aeaf52.png)

Reply:
Haha, whoops. Thanks.